index.bs

```diff
@@ -525,7 +527,10 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 6. Initialize |issuedRequests| to an empty list.
 
-7. For each authenticator currently available on this platform, perform the following steps:
+7. If no authenticator is available, reject |promise| with a <a>DOMException</a> whose name is "NotFoundError", and 
+    terminate this algorithm.
+
+8. For each authenticator currently available on this platform, perform the following steps:
```

Comment:
Note that Markdown is just as happy starting every numbered list item with "1.", so that new list items don't cause a diff for every subsequent item.